recipes/matplotlib-corefonts/LICENSE

```diff
@@ -0,0 +1,53 @@
+TrueType core fonts for the Web EULA
+
+
+END-USER LICENSE AGREEMENT FOR
+MICROSOFT SOFTWARE
+
+IMPORTANT-READ CAREFULLY: This Microsoft End-User License Agreement ("EULA") is a legal agreement between you (either an individual or a single entity) and Microsoft Corporation for the Microsoft software accompanying this EULA, which includes computer software and may include associated media, printed materials, and "on-line" or electronic documentation ("SOFTWARE PRODUCT" or "SOFTWARE"). By exercising your rights to make and use copies of the SOFTWARE PRODUCT, you agree to be bound by the terms of this EULA. If you do not agree to the terms of this EULA, you may not use the SOFTWARE PRODUCT.
+
+
+SOFTWARE PRODUCT LICENSE
+The SOFTWARE PRODUCT is protected by copyright laws and international copyright treaties, as well as other intellectual property laws and treaties. The SOFTWARE PRODUCT is licensed, not sold.
+
+
+1. GRANT OF LICENSE. This EULA grants you the following rights:
+
+    Installation and Use. You may install and use an unlimited number of copies of the SOFTWARE PRODUCT.
+    Reproduction and Distribution. You may reproduce and distribute an unlimited number of copies of the SOFTWARE PRODUCT; provided that each copy shall be a true and complete copy, including all copyright and trademark notices, and shall be accompanied by a copy of this EULA. Copies of the SOFTWARE PRODUCT may not be distributed for profit either on a standalone basis or included as part of your own product. 
```

Comment:
Looks like we can re-distribute it but I am not a lawyer.

Reply:
@ocefpaf Me neither, is there anybody in the conda-forge team who can help out with this or anybody with contact to microsoft.

Reply:
This is technically fine for us to redistribute, and I think we should have it as a package.  However, given some of the restrictions in the license, I (personally) would not recomend that we have any packages that depend on it.  I think people should explictly have to include it in their environment.